CONTRIBUTING.md

```diff
@@ -47,6 +62,26 @@ For more explained information about plugin override check [Development Override
 - Commit and push your changes to your branch then submit a pull request against the `master` branch.
 - A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
 
+#### PR Title Format
+We use [*Conventional Commits*](https://www.conventionalcommits.org/):
```

Comment:
knitpit: we really use conventional commits only for the commits in master that are normally derived from the PR title, commits in a PR can have any format and they'll be squashed when merged to master.

Reply:
Just to clarify this change was just moving the section from one place to another. I agree with what you mention, however the title does make reference to `PR Title Format` which I believe is correct.

Reply:
fair point